### PR TITLE
Mark rank and sub_searches as tech preview

### PR DESCRIPTION
**Pull request by jdconrad**
`rank` and `sub_searches` are in tech preview. This adds the tech preview text that is required in the docs for these features.

### CODE REVIEW

**Review by abdonpijpelink — COMMENTED** [review]
The inline admonition needs to have a single colon `:` or it won't render propertly.

File: docs/reference/search/search.asciidoc [hunk 1]
```diff
@@ -555,6 +555,7 @@ Period of time used to extend the life of the PIT.
 
 [[request-body-rank]]
 `rank`::
+preview::[]
```
[the remark this answers is not in the document]
Comment (jdconrad) [reply docs/reference/search/search.asciidoc]: Fixed.

**Comment by jdconrad** [comment]
@abdonpijpelink Thanks for the quick review. I have fixed the errors.

**Review by abdonpijpelink — APPROVED** [review]
LGTM

**Pull request merged**